test/pipelines/empty.test.js

```diff
@@ -0,0 +1,53 @@
+const supertest = require('supertest');
+const testHelper = require('../common/routing.helper');
+const config = require('../../lib/config');
+
+describe('Pipelines', () => {
+  describe('send a request that can\' be handled by any pipeline', () => {
```

Comment:
1) is it `can't` ? 
2) can we use "send a request that can't be handled by any pipeline"
so if you do a string search it will actually find a test ?

Reply:
Fixed.